src/main/java/ch/epfl/biop/bdv/command/exporter/BasicBdvViewToImagePlusExportCommand.java

```diff
@@ -121,13 +122,21 @@ public void run() {
                     .rangeT(selected_timepoints_str)
                     .sources(sourceList.toArray(new SourceAndConverter[0]))
                     .get();
```

Comment:
The get method calls `ImagePlusHelper.storeExtendedCalibrationToImagePlus` this is where we need to set the origin, and not in the command